### DISCUSSION TITLE
MotoTRBO Rest channel detection and scanning

### DISCUSSION THREAD

**Discussion by ryanthomp** [DSheirer/sdrtrunk#1448] [Ideas]
Like DSD there it can have the LSN stored and can scan for active rest channels for MotoTRBO Cap+ systems when the current frequency isn't the rest channel.

**Comment by DSheirer** [comment]
@ryanthomp are you asking to have a capability to scan outside of the defined channels to locate unknown rest channels that are being referred to during decode of known channels?

**Reply by ryanthomp** [reply]
No, when it loses a rest channel it scans known channels to find it and listen and monitor. Rather than A. Listening to every LCN in a system or B. scanning every LCN very fast usually missing which channel is the rest channel.
